### PR TITLE
add pandos-fignos and pandoc-xnos

### PR DESCRIPTION
**Pull request by basnijholt**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pandoc-fignos/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pandoc-xnos
+
+test:
+  commands:
+    - which pandoc-fignos
```
[the remark this answers is not in the document]
Comment (basnijholt) [reply recipes/pandoc-fignos/meta.yaml]: It is a pacdoc filter, so we would have to convert a file using pandoc and the filter.

[the remark this answers is not in the document]
Comment (scopatz) [reply recipes/pandoc-fignos/meta.yaml]: Well, is there a help message that you can check that includes the filter?

[the remark this answers is not in the document]
Comment (basnijholt) [reply recipes/pandoc-fignos/meta.yaml]: AKAIK there is no easy way. We could copy the relevant files from here https://github.com/tomduck/pandoc-fignos/blob/master/test/standard/Makefile

I don't know whether that's worth it though.

What do you think?

[the remark this answers is not in the document]
Comment (scopatz) [reply recipes/pandoc-fignos/meta.yaml]: I think it is worth having some kind of real test

**Pull request closed without merging**